demo/src/MemoryModels.tsx

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import { Button, Card, CardContent, TextField, Grid } from "@mui/material";
+
+interface MemoryModel {
+    isClass: boolean;
+    name: string;
+    id: string | number | null;
+    value: any;
+    stack_frame: boolean;
+    show_indexes: boolean;
+    style?: any[] | null;
+}
+
+function MemoryModels(props) {
+    const handleTextFieldChange = (event) => {
+        props.setFormData(event.target.value);
+    };
+
+    return (
+        <form onSubmit={props.onSubmit}>
+            <Card>
+                <CardContent>
+                    <Grid container spacing={2}>
+                        <Grid item xs={12}>
+                            <TextField
+                                id="multiline-textfield"
+                                label="Multi-line Text"
```

Comment:
Looks like this is also used as a placeholder text. Something like `Enter memory model JSON here`